### DISCUSSION TITLE
Can't use 32-bit plugins

### DISCUSSION THREAD

**Discussion by Jobsecond** [apohl79/audiogridder#396] [Help]
I have some old VST plugins which only have 32-bit version. Audiogridder failed to load them. 
I can use these plugins in 32-bit REAPER, so probably 32-bit Audiogridder server is required?
![Audiogridder failed to load 32-bit plugins](https://user-images.githubusercontent.com/25559254/108609863-be91c180-7385-11eb-96c6-6520d3556bcb.PNG)

**Comment by apohl79** [comment]
Yes AG has no 32 bit support.

**Reply by SQ80R** [reply]
Is there any plan to support 32bit Win VSTs on the Win AG server?

**Reply by apohl79** [reply]
nope ;-)

**Reply by SQ80R** [reply]
Hey Andreas, thanks for getting back so quick and I totally understand.
I'm currently using VEP at the moment and If it wasn't for the one pesky (but great) 32bit plug I use, I'd move over to AG in a heartbeat.

Love what you're doing with AG. Best regards from Scotland.

**Reply by emack999** [reply]
@SQ80R Get [jBridge ](https://jstuff.wordpress.com/jbridge/)and bridge that 32bit plugin into the 64 bit plugin folder.  Then you can use it in AG.

**Reply by SQ80R** [reply]
Thanks @emack999 I'll give that a go.

**Comment by JonnyHurricane** [comment]
Okay that saves me from asking that question. I also read a comment saying that a 32bit plugin would require a lot of code changes. Is this true for a 32bit server?

**Comment by nsputnik** [comment]
With so many 32bit plugins out there AG could be _THE_ only way to continue to use them with a modern DAW.
I have been wrestling with AG for 2 days trying to get it to work and I finally found this and it explains why. Please provide better docs for this project.

**Comment by nsputnik** [comment]
What does this feature on the homepage mean?
I'm pretty technical and even I thought this would mean it supports both 64 and 32-bit VSTs.
- 32/64 bit float processing

**Reply by apohl79** [reply]
It means that AG supports double precision (64bit) sample values (which only VST3 supports). You can enable this in hosts like Cubase.
